src/Benchmarks/Program.cs

```diff
@@ -50,6 +50,8 @@ public static void Main(string[] args)
                 var threads = GetThreadCount(config);
                 webHostBuilder = webHostBuilder.UseKestrel((options) =>
                 {
+                    options.UseHttps("testCert.pfx", "testPassword");
```

Comment:
Kestrel does a little bit of work creating a `ConnectionFilterContext` a `LibuvStream` and invoking an `HttpsConnectionFilter.OnConnectionAsync` which basically no-ops but still goes through the async state machine.

It's probably best not to call this in the benchmarks app if the URL scheme isn't HTTPS.